site.region

```diff
@@ -347,7 +347,7 @@
             </p>
           </div>
           <div class="site-footer__badgerIcon" title="Sally the Red Badger Badger">
-            <img src="/assets/badger-on-black.svg">
+            <img src="/assets/badger-on-black.svg" alt="Sally the badger image">
```

Comment:
Agree, @matt-paul would you mind changing this to remove 'image' from the description, please?